bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -27,102 +27,123 @@ <h2 id="payouts-section">Payouts</h2>
-        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
-      </tr>
-      <tr>
-        <th scope="row">SQL Injection</th>
-        <td>$6000</td>
-        <td>$3000</td>
-        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
+        <th scope="row">High <br>(sec-high)</th>
+        <td>$3000-$6000</td>
+        <td>$1000-$3000</td>
+        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a> - $500</td>
       </tr>
       <tr>
-        <th scope="row">CSRF<sup>3</sup></th>
-        <td>$5000</td>
-        <td>$2000</td>
-        <td>--</td>
-      </tr>
-      <tr>
-        <th scope="row">XSS<sup>4</sup></th>
-        <td>$5000</td>
-        <td>$2000</td>
+        <th scope="row">Moderate <br>(sec-moderate)</th>
+        <td>$1000-$3000</td>
+        <td>$500-$1000</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
       <tr>
-        <th scope="row">XXE</th>
-        <td>$5000</td>
-        <td>$2000</td>
+        <th scope="row">Low <br>(sec-low)</th>
+        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a> - $1000</td>
+        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a> - $500</td>
         <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
       </tr>
+    </tbody>
+  </table>
+
+  <ol class="mzp-u-list-styled">
+    <li>Excludes community websites</li>
+  </ol>
+
+  <p>Any bounty that receives a payout also obtains inclusion on our <a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">Hall of Fame</a>.</p>
+
+  <h2>Definition and Examples</h2>
+
+  <table id="payouts" class="mzp-u-data-table">
+    <thead>
       <tr>
-        <th scope="row">Domain Takeovers<sup>7</sup></th>
-        <td>$5000</td>
-        <td>$2000</td>
-        <td>$500<sup>5</sup></td>
-      </tr>
-      <tr>
-        <th scope="row">Server Side Request Forgery (SSRF)<sup>8</sup></th>
-        <td>$5000</td>
-        <td>$2000</td>
-        <td>--</td>
+        <th>Severity Rating</th>
+        <th>Definition and Examples</th>
       </tr>
+    </thead>
+    <tbody>
       <tr>
-        <th scope="row">XSS (minor)</th>
-        <td>$2000</td>
-        <td>$1000</td>
-        <td><a href="{{ url('security.bug-bounty.web-hall-of-fame') }}">HoF</a></td>
+        <th scope="row">Critical</th>
+        <td>
+          <p>
+            Critical vulnerabilities are urgent security issues that present an ongoing or immediate danger to the users of our services and our infrastructure
+            <ul class="mzp-u-list-styled">
+              <li>Remote Code Execution</li>
+              <li>Authentication and Session Management Flaws (which lead to account compromise)</li>
+              <li>Disclosure of secrets in publicly accessible assets</li>
+              <li>Hardcoded credentials for a privileged user</li>
+            </ul>
+          </p>
```

Comment:
Nit: this closing `</p>` tag should go before the list. The same for the other three below.

Reply:
done, thanks